sdk/iot/hub/samples/src/paho_iot_hub_methods_example.c

```diff
@@ -95,6 +96,7 @@ static az_result read_configuration_entry(
   return AZ_OK;
 }
 
+// Create mqtt endpoint e.g: ssl//contoso.azure-devices.net:8883
```

Comment:
Same nit here/elsewhere.
```suggestion
// Create MQTT endpoint e.g: ssl//contoso.azure-devices.net:8883
```